src/impl.cpp

```diff
@@ -1 +1,22 @@
 #include "impl.h"
+#include <iostream>
+#include <istream>
+#include <vector>
+#include <iterator>
+#include <sstream>
+#include <algorithm>
+#include <unordered_set>
+
+std::vector<std::string> uniqueWord(const std::string string)
+{
+    std::string text = std::move(string);
+ 
+    std::transform(text.begin(), text.end(), text.begin(), [](unsigned char c){ return std::tolower(c); });
+    std::istringstream iss(text);
+    std::vector<std::string> results((std::istream_iterator<std::string>(iss)), std::istream_iterator<std::string>());
```

Comment:
define an alias for std::istream_iterator and use it later
so it will look like this:

stream_iterator {iss},
stream_iterator {} 

And as I remember, you can just use implicit initialization of second param (replace for just braces {})